### ISSUE TITLE
Show message on deleted video page

### ISSUE THREAD

**Issue by Nutomic** [Chocobozzz/PeerTube#1443]
Imagine a video is deleted by the instance admin for some reason. If a user clicks on the video link from another website, it appears like the video was never there. Peertube doesn't show any indication that a video was deleted.

![screenshot_20181202_031205](https://user-images.githubusercontent.com/1044450/49334841-121a6800-f5e0-11e8-8946-e84eb6db5f64.png)

It would be good if we could show a delete reason here. This would be helpful to let people know why the video was deleted and maybe point to a discussion about it.

![screen-shot-2017-05-30-at-1 29 02-pm](https://user-images.githubusercontent.com/1044450/49334896-45112b80-f5e1-11e8-825c-a4aff32f4318.png)

Basically a similar design to this, but showing the delete reason instead.

**Comment by Chocobozzz** [comment]
Closing now we have a dedicated 404 page
